docs/Installation.md

```diff
@@ -127,19 +141,19 @@ The user can now see all the configured data points for this asset, as shown in
 - Tank Temperature in celsius
 - Tank level in liters
 
-![machine_monitor](graphics/Machine_Monitor_Configured_Datapoints.png)
+![Machinemonitortags.PNG](graphics/Machinemonitortags.PNG)
 
 The Machine Monitor example utilize these acquired data points to create maintenance rules and templates to govern the state and health of the process. The first rule states that when the number of produced bottles exceeds a certain number, a maintenance routine has to be carried out. A second rule can be created for the number of faulty bottles, where if the value of the variable exceeds a certain threshold, the process has to be stopped and inspected. Third and fourth rules can be created for the temperature and level of the tank to ensure that the process is running correctly and the system is free from any issues.
 
 ### Connectivity troubleshooting
 
-Note: If the configured parameters do not appear in the Machine Monitor application, the user needs to check the configuration of the Data Service application and then verify the credentials used for connecting the Machine Monitor application to the Databus. To do this, click on the "Settings" tab in the sidebar and then select "Connectivity Configuration," as highlighted in the figure below:
+Note: If the configured parameters do not appear in the Machine Monitor application, the user needs to check the configuration of the IIH Essentials application and then verify the credentials used for connecting the Machine Monitor application to the Databus. To do this, click on the "Settings" tab in the sidebar and then select "Connectivity Configuration," as highlighted in the figure below:
 
-![machine_monitor](graphics/Machine_Monitor_Connectivity-Config.png)
+![Machinemonitorconfigsettings](graphics/Machinemonitorconfigsettings.PNG)
```

Comment:
This picture should be switched with the one after, the order is not correct

Reply:
I swapped the pictures